Issuing a random INTERNALCMD on port 25 crashes the whole cluster

### system info
Haraka | Haraka.js — Version: 2.8.20 (pulled from git)
 --- | :--- 
Node | v10.6.0
OS | Linux out-mta01 4.16.0-0.bpo.2-amd64 #1 SMP Debian 4.16.16-2~bpo9+1 (2018-06-26) x86_64 GNU/Linux
openssl | OpenSSL 1.1.0f  25 May 2017

### Expected behavior
Reply with something (command not found, missing key?) or nothing, but at least do not crash the whole instance (master process too) (cluster mode)

### Observed behavior
2018-07-15T22:50:21.427Z [ERROR] [-] [core] Invalid command: 89c68114f5edc7bb786b78a411ba0e8daa65697b347d95c660a8bfba826d2977
2018-07-15T22:50:21.427Z [CRIT] [-] [core] TypeError: Cannot read property 'apply' of undefined
2018-07-15T22:50:21.427Z [CRIT] [-] [core]     at Object.Server.receiveAsMaster (/opt/haraka/server.js:242:21)
2018-07-15T22:50:21.427Z [CRIT] [-] [core]     at EventEmitter.messageHandler (/opt/haraka/server.js:254:16)
2018-07-15T22:50:21.427Z [CRIT] [-] [core]     at EventEmitter.emit (events.js:182:13)
2018-07-15T22:50:21.427Z [CRIT] [-] [core]     at Worker.<anonymous> (internal/cluster/master.js:177:13)
2018-07-15T22:50:21.428Z [CRIT] [-] [core]     at Worker.emit (events.js:187:15)
2018-07-15T22:50:21.428Z [CRIT] [-] [core]     at ChildProcess.Worker.process.on (internal/cluster/worker.js:28:12)
2018-07-15T22:50:21.428Z [CRIT] [-] [core]     at ChildProcess.emit (events.js:182:13)
2018-07-15T22:50:21.428Z [CRIT] [-] [core]     at emit (internal/child_process.js:811:12)
2018-07-15T22:50:21.428Z [CRIT] [-] [core]     at process._tickCallback (internal/process/next_tick.js:63:19)

### Steps to reproduce
While trying to understand how to flush the queue using INTERNALCMD, i was able to instantanly crash the whole haraka daemon.

telnet localhost 25
Trying ::1...
Connected to localhost.
Escape character is '^]'.
220 some.fancy.prompt ESMTP Haraka ready
INTERNALCMD somekey help
250 Command sent for execution. Check Haraka logs for results.
Connection closed by foreign host.

Maybe just an "easy" way to avoid this. If i'm not wrong the syntax is:

`INTERNALCMD <secretkey> <command and stuff>` ?

In that cas could it be possible to filter out attemps by first verifying that the key is valid ? And politely reject if it doesn't match.

My tests shows that even with an invalid key the thing crashes.

```
# telnet localhost 25
Trying ::1...
Connected to localhost.
Escape character is '^]'.
220 out-crashme.somedomaine.com ESMTP Haraka ready
INTERNALCMD caca
250 Command sent for execution. Check Haraka logs for results.
Connection closed by foreign host.
```

```
2018-07-21T07:58:51.854Z [NOTICE] [71794746-D611-43E5-A5D4-53F18F9C6053] [core] connect ip=::1 port=57278 local_ip=:: local_port=25
2018-07-21T07:58:56.191Z [ERROR] [-] [core] Invalid command: caca
2018-07-21T07:58:56.192Z [CRIT] [-] [core] TypeError: Cannot read property 'apply' of undefined
2018-07-21T07:58:56.192Z [CRIT] [-] [core]     at Object.Server.receiveAsMaster (/opt/haraka/server.js:242:21)
2018-07-21T07:58:56.192Z [CRIT] [-] [core]     at EventEmitter.messageHandler (/opt/haraka/server.js:254:16)
2018-07-21T07:58:56.192Z [CRIT] [-] [core]     at EventEmitter.emit (events.js:182:13)
2018-07-21T07:58:56.193Z [CRIT] [-] [core]     at Worker.<anonymous> (internal/cluster/master.js:177:13)
2018-07-21T07:58:56.193Z [CRIT] [-] [core]     at Worker.emit (events.js:187:15)
2018-07-21T07:58:56.193Z [CRIT] [-] [core]     at ChildProcess.Worker.process.on (internal/cluster/worker.js:28:12)
2018-07-21T07:58:56.193Z [CRIT] [-] [core]     at ChildProcess.emit (events.js:182:13)
2018-07-21T07:58:56.193Z [CRIT] [-] [core]     at emit (internal/child_process.js:811:12)
2018-07-21T07:58:56.193Z [CRIT] [-] [core]     at process._tickCallback (internal/process/next_tick.js:63:19)
```
I'm not willing to bug you with this, just trying to find a way to cleanly handle this.

--- a/server.js
+++ b/server.js
@@ -238,6 +238,7 @@ Server.sendToMaster = function (command, params) {
 Server.receiveAsMaster = function (command, params) {
     if (!Server[command]) {
         logger.logerror(`Invalid command: ${command}`);
+        return;
     }
     Server[command].apply(Server, params);
 }

